ui/admin/app/templates/scopes/scope/credential-stores/index.hbs

```diff
@@ -6,8 +6,11 @@
   <page.header>
     <h2>
       {{t 'resources.credential-store.title_plural'}}
+      <DocLink @doc='credential-stores' @iconSize='large' />
```

Comment:
ATM this does not redirect-properly. It could be a problem on the `boundary/website`.